### PR TITLE
apply iframe UI

### PR DESCRIPTION
**Pull request by fanlan1210**
* with basic user overlay (can toggle show/hide state)

### CODE REVIEW

File: sites/game-client/client.ejs [hunk 1]
```diff
@@ -216,7 +216,11 @@ absolute URL.
       import AvatarSelectionPage from '/static/sites/game-client/ui/avatar-selection.mjs';
       import OverlayPosition from '/static/sites/game-client/ui/overlay-position.mjs';
       import ErrorModal from '/static/sites/game-client/ui/error-modal.mjs';
+<<<<<<< HEAD
```
Comment (john0312) [inline sites/game-client/client.ejs:219]: You might have to resolve this.

File: extensions/iframe/common/client.mjs [hunk 2]
```diff
@@ -14,7 +14,8 @@ class IframeOverlay extends Overlay {
 
     const expandBtn  = document.getElementById('iframe-expand');
     expandBtn.onclick = () => {
-      mainUI.enterFocusMode(this, OverlayPosition.LEFT_BOTTOM);
```
Comment (john0312) [inline extensions/iframe/common/client.mjs:17]: Maybe add a TODO to note on better handling for hiding of userOverlay. Or just modify `enterFocusMode()` to handle hiding of userOverlay?

File: sites/game-client/ui/user-overlay.mjs [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+   * @constructor
+   * @param {MainUI} mainUI - The MainUI object.
+   */
+  constructor(mainUI) {
+    const dom = document.getElementById(USER_OVERLAY_DIV);
+    super(mainUI, dom);
+  }
+
+  show() {
+    $(this.dom).show();
```
Comment (john0312) [inline sites/game-client/ui/user-overlay.mjs:19]: Maybe add a TODO on correctly implementing the Overlay. i.e. add corresponding OverlayPosition

Comment (fanlan1210) [reply sites/game-client/ui/user-overlay.mjs]: added.

**Review by john0312 — APPROVED** [review]

**Pull request merged**